client/src/main/kotlin/no/nav/su/se/bakover/client/oppgave/OppgaveHttpClient.kt

```diff
@@ -135,7 +135,7 @@ internal class OppgaveHttpClient(
                                 behandlingstema = config.behandlingstema?.toString(),
                                 behandlingstype = config.behandlingstype.toString(),
                                 aktivDato = aktivDato,
-                                fristFerdigstillelse = aktivDato.plusDays(30),
+                                fristFerdigstillelse = config.fristFerdigstillelse ?: aktivDato.plusDays(30),
```

Comment:
Trenger vi en default her?

Reply:
Vi har hatt som default 30 dagar frist tidligere, så behåller det. 😊

Reply:
Hadde føltes tryggere med non-nullables også definerte de spesfikke contextene det hver for seg.